esignet-insurance.properties

```diff
@@ -508,9 +508,9 @@ mosip.esignet.vci.key-values={\
                   'policyName': {'display': {{'name': 'Policy Name','locale': 'en'}}},\
                   'policyNumber': {'display': {{'name': 'Policy Number','locale': 'en'}}}\
             }},\
-          'display': {{'name': 'Sunbird Insurance', \
+          'display': {{'name': 'Veridonia Insurance', \
                         'locale': 'en', \
-                        'logo': {'url': 'https://sunbird.org/images/sunbird-logo-new.png','alt_text': 'a square logo of a Sunbird'},\
+                        'logo': {'url': 'https://${mosip.api.public.host}/inji/veridonia-logo.png','alt_text': 'a square logo of a Veridonia'},\
                         'background_color': '#FDFAF9',\
                         'text_color': '#7C4616'}},\
           'order' : {'fullName','policyName','policyExpiresOn','policyIssuedOn','policyNumber','mobile','dob','gender','benefits','email'}\
```

Comment:
Also I don't see the changes related to the Term and Life insurance. Is that planned for later ?

Reply:
we will add it later @vishwa-vyom , once clarified from @holash